src/components/HomePage.js

```diff
@@ -57,7 +61,13 @@ function HomePage({ user, setUser }) {
         </div>
       )}
 
-      <div>{!authenticated ? <Login /> : <TaskView userID={user._id} />}</div>
+      <div>
+        {!authenticated ? (
+          <Login />
+        ) : (
```

Comment:
This is quite a disgusting line of code.

Reply:
JSX code that has lost the will to live